keyboards/arabica37/info.json

```
@@ -0,0 +1,65 @@
+{
+    "keyboard_name": "arabica37 rev. 1",
+    "url": "",
+    "maintainer": "qmk",
+    "width": 15.25,
+    "height": 5.75,
+    "layouts": {
+        "LAYOUT": {
+            "layout": [
+                {"label":"Esc", "x":0.25, "y":0.75},
+                {"label":"Q", "x":1.25, "y":0.75},
+                {"label":"W", "x":2.25, "y":0.25},
+                {"label":"E", "x":3.25, "y":0},
+                {"label":"R", "x":4.25, "y":0.25},
+                {"label":"T", "x":5.25, "y":0.5},
+
+                {"label":"Y", "x":9, "y":0.5},
+                {"label":"U", "x":10, "y":0.25},
+                {"label":"I", "x":11, "y":0},
+                {"label":"O", "x":12, "y":0.25},
+                {"label":"P", "x":13, "y":0.75},
+                {"label":"Backspace", "x":14, "y":0.75},
+
+                {"label":"Ctrl", "x":0.25, "y":1.75},
+                {"label":"A", "x":1.25, "y":1.75},
+                {"label":"S", "x":2.25, "y":1.25},
+                {"label":"D", "x":3.25, "y":1},
+                {"label":"F", "x":4.25, "y":1.25},
+                {"label":"G", "x":5.25, "y":1.5},
+
+                {"label":"H", "x":9, "y":1.5},
+                {"label":"J", "x":10, "y":1.25},
+                {"label":"K", "x":11, "y":1},
+                {"label":"L", "x":12, "y":1.25},
+                {"label":";", "x":13, "y":1.75},
+                {"label":"'", "x":14, "y":1.75},
+
+                {"label":"Adjust", "x":0.25, "y":2.75},
+                {"label":"Z", "x":1.25, "y":2.75},
+                {"label":"X", "x":2.25, "y":2.25},
+                {"label":"C", "x":3.25, "y":2},
+                {"label":"V", "x":4.25, "y":2.25},
+                {"label":"B", "x":5.25, "y":2.5},
+
+                {"label":"N", "x":9, "y":2.5},
+                {"label":"M", "x":10, "y":2.25},
+                {"label":",", "x":11, "y":2},
+                {"label":".", "x":12, "y":2.25},
+                {"label":"/", "x":13, "y":2.75},
+                {"label":"\\", "x":14, "y":2.75},
+
+                {"label":"Shift", "x":0, "y":3.75, "w":2.25},
+                {"label":"Lower", "x":4.375, "y":3.75, "w":1.5},
+                {"label":"GUI", "x":5.875, "y":3.75, "w":1.25},
+
+                {"label":"Alt", "x":8.125, "y":3.75, "w":1.25},
+                {"label":"Raise", "x":9.375, "y":3.75, "w":1.5},
+                {"label":"Shift", "x":13, "y":3.75, "w":2.25},
+
+                {"label":"Space", "x":4.875, "y":4.75, "w":1.25},
+                {"label":"Enter", "x":8.625, "y":4.75, "w":1.25}
```

I messed up the positioning on my suggestion; this fixes it.

```suggestion
                {"label":"Space", "x":5.125, "y":4.75, "w":1.25},
                {"label":"Enter", "x":8.875, "y":4.75, "w":1.25}
```